specification/maps/data-plane/Search/preview/1.0/search.json

```diff
@@ -1839,7 +1839,63 @@
         },
         "geometryData": {
           "description": "Geometry data in GeoJSON format. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946) for details. Present only if \"error\" is not present.",
-          "$ref": "../../../Common/preview/1.0/common.json#/definitions/GeoJsonObject"
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/GeoJsonFeatureCollection"
+        }
+      }
+    },
+    "GeoJsonPolygonFeature": {
+      "description": "A valid `GeoJSON Polygon` feature type. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.1.6) for details.",
+      "x-ms-discriminator-value": "Feature",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/GeoJsonFeature"
+        }
+      ],
+      "properties": {
+        "geometry": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/GeoJsonPolygon"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties can contain any additional metadata about the `Feature`. Value can be any JSON object or a JSON null value",
+          "type": "object"
+        }
+      },
+      "required": [
+        "geometry",
+        "properties"
+      ]
+    },
+    "GeoJsonMultiPolygonFeature": {
```

Comment:
The definition should be the same as `GeoJsonPolygonFeature` except the `description` and `properties:geometry` parts.